src/style.css

```diff
@@ -1,3 +1,81 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts. It's good practice to have a fallback font in case the primary font isn't available. Consider adding a more generic font family as a backup after 'Roboto'. For example, 'font-family: Roboto, Arial, sans-serif;' could be 'font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;'.